api/api.proto

```diff
@@ -179,6 +179,18 @@ enum OperationFlags {
   OP_FLAGS_DETACH_FORCE = 2;
 }
 
+enum VolumePlacementEnforcementType {
```

Comment:
Need comments for the enum type and its values.

Reply:
Also move this to `VolumePlacementRule` so that the scope namespace is set automatically by the generator.

Reply:
Done